inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -529,4 +323,51 @@ void MKLDNNSplitNode::setDynamicBatchLim(int lim) {
         prim.setBatchLimit(batchToProcess(), getParentEdges().size(), getChildEdges().size());
     }
 }
+
+inline uint8_t* MKLDNNSplitNode::getDataPtr(const MKLDNNMemory& memoryPtr) {
+    return reinterpret_cast<uint8_t*>(memoryPtr.GetData()) + memoryPtr.GetDescriptor().data.layout_desc.blocking.offset_padding *
+            MKLDNNExtensionUtils::sizeOfDataType(mkldnn::memory::data_type(memoryPtr.GetDescriptor().data.data_type));
+}
+
+void MKLDNNSplitNode::prepareOptimizedParams() {
+    auto inpTensorDesc = this->getSelectedPrimitiveDescriptor()->getConfig().inConfs[0].desc;
+
+    //find axis order position
+    auto order = inpTensorDesc.getBlockingDesc().getOrder();
+    unsigned axisOrderPos = UINT_MAX;
+    for (size_t i = 0; i < order.size(); ++i) {
+        if (order[i] == axis) {
+            axisOrderPos = i;
+            break;
+        }
+    }
+    if (UINT_MAX == axisOrderPos) {
+        THROW_IE_EXCEPTION << "Can't find the axis in the input tensor order list";
```

Comment:
Please use unified way to write error messages though all the node (for example, https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/cum_sum.cpp#L30). I have heard a lot of complains that it is impossible to understand which particular layer is broken using our error messages.

Reply:
Done in ee8234d71967059ae0d72418abd7796120119305.